### PR TITLE
Rotwing guidance bank fix

### PR DESCRIPTION
**Pull request by Dennis-Wijngaarden**
- Limit lateral acceleration in guidance_indi_hybrid
- Fix bank angle in WLS outer loop(s)

@EwoudSmeur, please verify

### CODE REVIEW

File: sw/airborne/modules/checks/preflight_checks.c [hunk 1]
```diff
@@ -43,8 +43,13 @@
 #define PREFLIGHT_CHECK_INFO_TIMEOUT 5
 #endif
 
-static struct preflight_check_t *preflight_head = NULL;
+#ifdef PREFLIGHT_CHECK_BYPASS
+bool preflight_bypass = PREFLIGHT_CHECK_BYPASS;
+#else
 bool preflight_bypass = FALSE;
+#endif
```
Comment (gautierhattenberger) [inline sw/airborne/modules/checks/preflight_checks.c:50]: For coherence, can it be written like this ?
```suggestion
#ifndef PREFLIGHT_CHECK_BYPASS
#define PREFLIGHT_CHECK_BYPASS FALSE
#endif

bool preflight_bypass = PREFLIGHT_CHECK_BYPASS;
```

File: sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c [hunk 2]
```diff
@@ -118,7 +118,7 @@ struct FloatMat33 Ga_inv;
 struct FloatVect3 control_increment; // [dtheta, dphi, dthrust]
 
 float filter_cutoff = GUIDANCE_INDI_FILTER_CUTOFF;
-float guidance_indi_max_bank = GUIDANCE_H_MAX_BANK;
+float guidance_indi_max_bank = DegOfRad(GUIDANCE_H_MAX_BANK);
```
Comment (gautierhattenberger) [inline sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c:121]: I don't see why this should be converted to degrees, to convert it again to radians later

**Review by gautierhattenberger — DISMISSED** [review]

File: sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c (hunk 2 above)
Comment (dewagter) [reply sw/airborne/firmwares/rotorcraft/guidance/guidance_indi.c:121]: The setting in the GCS is otherwise in radians.

**Comment by gautierhattenberger** [comment]
You have `unit` and `alt_uni` attributes for settings to tell how it should be displayed and sent from the GCS

**Comment by dewagter** [comment]
PR updated!

**Comment by EwoudSmeur** [comment]
Thanks @dewagter and @gautierhattenberger for looking into this, didn't know the setting could also be configured with an alt_unit. Like this it works!

**Comment by gautierhattenberger** [comment]
Some stuff seem to be merged with #3274 already. Can you rebase on master to check ? Or we just believe in git...

**Comment by dewagter** [comment]
Rebased on master works fine.

**Review by dewagter — APPROVED** [review]
Bypass of preflight in NPS is OK.
Setting MAX_BANK is now done the clean way.
Guidance bank bug is resolved.

**Pull request merged**